plugin/ale/cursor.vim

```diff
@@ -72,7 +72,7 @@ function! ale#cursor#TruncatedEcho(message)
 
     try
         " Echo the message truncated to fit without creating a prompt.
-        set shortmess+=T
+        setl shortmess+=T
```

Comment:
I wonder if we can get a `Vint` configuration setting to check for that, or something.